project/Dependencies.scala

```diff
@@ -17,9 +17,9 @@ object Dependencies {
   val monocleV = "2.0.0"
 
   val workbenchUtilV = "0.5-4c7acd5"
-  val workbenchModelV = "0.13-6dc016b"
+  val workbenchModelV = "0.13-db13244-SNAP"
   val workbenchGoogleV = "0.21-890a74f"
-  val workbenchGoogle2V = "0.6-c91d96b"
+  val workbenchGoogle2V = "0.6-8ad0760-SNAP"
```

Comment:
this seems bad....

Reply:
see PR description

Reply:
I think we should change to non-SNAP before merge

Reply:
I'd love to..but I don't know when Doug will have time for wb-libs PR...yet I'd like to push this to dev asap to help debug problem with subcriber

Reply:
Maybe ping him on the PR? Seems you resolved his comment. We can merge with a SNAP to resolve the issue but we shouldn't stay on a SNAP forever.